core/storage/feedback/gae_models.py

```diff
@@ -287,3 +287,15 @@ def get_by_exploration_and_thread_id(cls, exploration_id, thread_id):
         Returns None if it doesn't match anything."""
 
         return cls.get_by_id(cls._get_instance_id(exploration_id, thread_id))
+
+
+class FeedbackMessageEmailDataModel(base_models.BaseModel):
+    """Instance of this model feedback message detail for sending email later.
+
+    This model instance is created per user with id set to user_id"""
+
+    # This contains exploraion_id, thread_id and message_id which is used to get
+    # instance of FeedbackMessageModel.
+    feedback_messages = ndb.JsonProperty(repeated=True, indexed=True)
+    # Retires count for sending an email.
```

Comment:
The number of failed attempts to send an email to this user that have been made so far.